src/View/Helper/I18nHelper.php

```diff
@@ -239,6 +239,17 @@ public function reset(): void
      */
     private function getTranslatedField(array $object, string $attribute, string $lang, array &$included): ?string
     {
+        // first look if embedded relationships are set
+        if (!empty($object['relationships']['translations']['data'][0]['attributes'])) {
```

Comment:
it's ok. However I'd rather use `Hash::check`:
```
if (!Hash::check($object, 'relationships.translations.data.0.attributes')) {
```

Reply:
Right, done!